src/ScoreCard.js

```diff
@@ -0,0 +1,77 @@
+function ScoreCard() {
+  this.frame = 1;
+  this.pins = 10;
+  this.roll = 1;
+  this.scores = {1:[], 2:[], 3:[], 4:[], 5:[], 6:[], 7:[], 8:[], 9:[], 10:[]};
```

Comment:
It seems like you're getting a vanilla js object to behave like an array here. Why not just use an array?